lib/ts/itinerary.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Itinerary do
+	@moduledoc """
+	Defines the operations that can be performed on the itinerary container.
+	"""
```

Comment:
a single line comment detailing the purpose of each method would be helpful.

Reply:
That has been done for the Itinerary module.